tests/callbacks/test_xla_stats_monitor.py

```diff
@@ -69,7 +77,7 @@ def test_xla_stats_monitor_no_tpu_warning(tmpdir):
     model = BoringModel()
     xla_stats = XLAStatsMonitor()
 
-    trainer = Trainer(default_root_dir=tmpdir, callbacks=[xla_stats], max_steps=1, tpu_cores=None)
+    trainer = Trainer(default_root_dir=tmpdir, callbacks=[xla_stats], max_steps=1, devices=None)
```

Comment:
```suggestion
    trainer = Trainer(default_root_dir=tmpdir, callbacks=[xla_stats], max_steps=1)
```

Isn't this the same?

Reply:
I think this test doesn't make sense anymore, this scenario is not equally reproducible by `accelerator=` and `device=` 

This actually equal to 
`trainer = Trainer(default_root_dir=tmpdir, callbacks=[xla_stats], max_steps=1, device=None, accelerator="tpu")
But current accelerator_connector logic could handle it, if device =None, the device it will been set by accelerator_type(in `_set_devices_if_none`), and accelerator_type set by select_accelerator() which will check either accelerator flag or TPU availability. So the it won't be a Misconfig as the training will run on TPU.

Reply:
Yeah, there seem to be some tests that won't `Misconfigure` when only using `devices` and `accelerator`. Should I write up a new test to get a Misconfiguration warning? Or should we delete these tests?

Reply:
I'm leaning towards deleting the test entirely.

Reply:
I think we can delete it

Reply:
nuke it. the name and description of the test is anyway lying.